influence_benchmark/stats/preferences_per_iteration.py

```diff
@@ -65,10 +66,27 @@ def compute_iteration_statistics(traj_df: pd.DataFrame, top_n: int) -> Dict[str,
     traj_df_filtered = filter_traj_df(traj_df, num_chosen_trajs=top_n, func=pd.DataFrame.nlargest)
     subenv_df = group_traj_df_to_subenv_df(traj_df, traj_df_filtered)
 
+    num_trajs = len(traj_df["traj_rew"])
+    num_filtered_trajs = len(traj_df_filtered["traj_rew"])
+
     results["rew_avg_all_trajs"] = subenv_df["mean_traj_reward"].mean()
+    results["rew_stderr_all_trajs"] = (np.std(traj_df["traj_rew"], ddof=1) / np.sqrt(num_trajs)) if num_trajs > 1 else 0
```

Comment:
I think we should probably have a util for this. I usually write a `mean_and_std_err` util in other projects

Reply:
Also would we ever pass num_trajs = 0? Maybe we should just have an assert that it's non-zero?

Reply:
Don't think we'd ever pass empty array but the computation fails when num_traj = 1 cuz of the ddof=1. This is cuz it's the sample std, which needs the array to have multiple elements.